src/modules/endpoint/reducers/__tests__/cityContentReducer.spec.js

```diff
@@ -85,7 +85,11 @@ describe('cityContentReducer', () => {
   ]
   for (const action of unsupportedActionsOnUnitializedState) {
     it(`should throw on ${action.type} if state is unitialized`, () => {
-      expect(() => cityContentReducer(null, action)).toThrow()
+      if (action === pushCategoryAction || action === pushEventAction) {
+        expect(cityContentReducer(null, action)).toBeNull()
```

Comment:
Shouldn't the same apply for most actions that threw errors before? So for example also for morphContentLanguage, etc.? @maxammann

Reply:
Not sure what you're referencing here. Do you  mean the test should be similar for other reducers?

Reply:
I think he means that there are still some actions for which the cityContentReducer throws if the state is null. This should probably also be changed to just return null.